src/Routes/RosPage/RosPage.js

```diff
@@ -124,7 +130,46 @@ class RosPage extends React.Component {
         ).then((results) => results);
     }
 
+    updateStateFilter = (value) => {
+        this.setState({
+            stateFilterValue: value
+        });
+    }
+
+    onDeleteFilters = (e, filtersArr) => {
+        const deletedStateFilters = filtersArr.filter((filterObject) => {
+            return filterObject.category === 'State';
+        });
+
+        if (deletedStateFilters.length > 0) {
+            const resetFiltersList = deletedStateFilters[0]?.chips.map((chip) =>{
+                return chip?.name;
+            });
+
+            const activeStateFilters = this.state.stateFilterValue.filter(filterName => !resetFiltersList.includes(filterName));
+
+            this.setState ({
+                stateFilterValue: activeStateFilters
+            });
+        }
+    }
+
+    getActiveFilterConfig = () => {
+        let activeFilters = this.state.stateFilterValue.map((value)=> {
+            return { name: value };
+        });
```

Comment:
No need to use let, since we are not mutating the value. Also extra return statement
```suggestion
        const activeFilters = this.state.stateFilterValue.map((value)=> ({ name: value }));
```

Reply:
Thank you for the suggestion @karelhala updated the code with same.